gpu-enabler/src/main/scala/com/ibm/gpuenabler/CUDARDDUtils.scala

```diff
@@ -352,7 +438,6 @@ object CUDARDDImplicits {
           }
         }
       }
-
```

Comment:
Can we revert this since this change is not necessary?

Reply:
Yes. No ploblem.
Sorry to make trivial mistakes.